addons/mrp/models/stock_move.py

```diff
@@ -566,7 +604,8 @@ def _compute_kit_quantities(self, product_id, kit_qty, kit_bom, filters):
                     continue
                 incoming_moves = bom_line_moves.filtered(filters['incoming_moves'])
                 outgoing_moves = bom_line_moves.filtered(filters['outgoing_moves'])
-                qty_processed = sum(incoming_moves.mapped('product_qty')) - sum(outgoing_moves.mapped('product_qty'))
+                qty_field = "quantity" if "qty_done" in self.env.context else 'product_qty'
```

Comment:
I would use the move state to know which 'quantity' to take

Reply:
already resloved in https://github.com/odoo/odoo/pull/164544